css-view-transitions-1/Overview.bs

```diff
@@ -1250,18 +1272,17 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 		1. Let |document| be |transition|'s [=relevant global object's=] [=associated document=].
 
-		1. Remove all associated [=view-transition pseudo-elements=] from |document|.
-
-				Issue: There needs to be a definition/link for "remove".
-
-				Issue: There needs to be a definition/link for "associated".
+		1. [=Assert=]: |document|'s [=document/active DOM transition=] is |transition|.
 
 		1. [=list/For each=] |capturedElement| of |transition|'s [=ViewTransition/named elements=]' [=map/values=]:
 
 			1. [=list/For each=] |style| of |capturedElement|'s [=captured element/style definitions=]:
 
 				1. If |style| is not null,
+					and |style| is in |document|'s [=document/view transition style sheet=],
 					then remove |style| from |document|'s [=document/view transition style sheet=].
+
+		1. Set |document|'s [=document/active DOM transition=] to null.
```

Comment:
Shouldn't we reset [=show view-transition root pseudo-element=] here? Otherwise it starts of as true and the next time [=document/active DOM transition=] is set, it's ::view-transition is displayed before Setup View transition.